docs: Update MCP tools reference (#4822)
n8n-io/n8n-docs: docs/advanced-ai/mcp/mcp_tools_reference.md

| `data[].scopes` | `string[]` | User permissions for this workflow |
| `data[].canExecute` | `boolean` | Whether the user has permission to execute this workflow |
| `data[].availableInMCP` | `boolean` | Whether the workflow is visible to MCP tools |
| `data[].tags` | `array` | Tags assigned to the workflow, each with `id` and `name` |
| `count` | `integer` | Total number of workflows that match the filters |

#### Notes

- Maximum result limit is 200.
- Results are sorted by most recently updated workflows first by default.
- Includes user permission scopes for each workflow so MCP clients can see what actions are available for the workflow.

> - Filtering by `tags` and the `tags` field in results are available from n8n v2.27.0. Use `list_tags` to discover the available tag names.

Reviewer: ```suggestion
- Filtering by `tags`, and the `tags` field in results, are available from n8n v2.27.0. Use `list_tags` to discover the available tag names.
```

| `data[].type` | `"personal" | "team"` | The project type |
| `data[].matchType` | `"exact" | "partial"` | Whether the project name matches the query exactly or partially. Only present when `query` is provided |
| `count` | `integer` | Total number of matching projects |

> | `teamProjectsEnabled` | `boolean` | Whether team projects are licensed on this instance. When `false`, default to omitting `projectId` on `create_workflow_from_code` so the workflow lands in the caller's personal project, unless the user explicitly picked one of the returned accessible projects. Omitted on error responses. Available from n8n v2.26.0. |

Reviewer: ```suggestion
| `teamProjectsEnabled` | `boolean` | Whether team projects are licensed on this instance. When `false`, `projectId` is omitted by default on `create_workflow_from_code`, so the workflow is created in the caller's personal project, unless the user explicitly selects one of the returned accessible projects. Omitted on error responses. Available from n8n v2.26.0. |
```
